src/go/supervisor/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/rancher-sandbox/rancher-desktop/src/go/supervisor
+
+go 1.18
+
+require (
+	github.com/spf13/cobra v1.5.0
+	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418
+)
+
+require (
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
This should be the indirect section — please run `go mod tidy` and see if there's any difference.